packages/eslint-plugin/docs/Testing.md

```diff
@@ -0,0 +1,32 @@
+# Testing
+
+## Introduction
+
+This document provides information about how to test the eslint-plugin after adding or removing rules.
+
+## Example-app
+
+The example-app folder is located at the root of `react-native-project-config`.
+The goal of each file of example-app is to simulate the behavior of a rule or a set of rules, that you can visually check it or by running the `yarn lint` command.
```

Comment:
"so that you can visually check it" plutôt ? Ou alors "that you can visually check" mais j'ai l'impression que c'est l'un ou l'autre #AnglaisLV1

Reply:
Yes en effet !